source/code/scxcorelib/pal/scxlibglob.cpp

```diff
@@ -22,12 +22,12 @@ namespace SCXCoreLib
         // if directory paths are non-existant, provide default system library paths
         if (paths.size() == 0)
         {
-            m_paths.push_back(L"/lib");
-            m_paths.push_back(L"/usr/lib");
 #if PF_WIDTH == 64
             m_paths.push_back(L"/lib64");
             m_paths.push_back(L"/usr/lib64");
```

Comment:
Ubuntu doesn't put libraries there, it puts them in `/lib/x86_64-linux-gnu/`. I know we don't have platform specific code, but perhaps this should be in the path as well for 64-bit?